allensdk/brain_observatory/vbn_2022/metadata_writer/metadata_writer.py

```diff
@@ -134,8 +162,30 @@ def run(self):
                     index_col='ecephys_session_id',
                     on_missing_file=self.args['on_missing_file'])
 
-        session_table.to_csv(self.args['ecephys_sessions_path'],
-                             index=False)
-        behavior_session_table.to_csv(
-                             self.args['behavior_sessions_path'],
-                             index=False)
+        self.write_df(
+            df=session_table,
+            output_path=self.args['ecephys_sessions_path'])
+
+        self.write_df(
+            df=behavior_session_table,
+            output_path=self.args['behavior_sessions_path'])
+
+        output_data = dict()
```

Comment:
Please write this as a single dict literal. It is easier to see what is being stored in this variable that way.

Reply:
Done.